src/styles/card.css

```diff
@@ -0,0 +1,106 @@
+.card {
+  display: flex;
+  width: 200px;
+  flex-direction: column;
+  padding: 32px 16px 16px;
+  margin: auto;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 7px;
+  justify-content: center;
+  align-items: center;
+}
+
+.card__image {
+  height: 134px;
+  width: 160px;
+}
+
+.card__title {
+  font-style: normal;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  color: #060b35;
+  margin-top: 40px;
+  margin-bottom: 0;
```

Comment:
Just reset default browser margins. You have already made it with `body`. So you can add `p` and `h` tags as well